test/cypress/integration/basic/initialization.spec.js

```diff
@@ -0,0 +1,53 @@
+describe('Editor.js with default configuration', () => {
+  it('should have a global EditorJS function', () => {
+    cy.visit(Cypress.env('local_test_server_url'));
+
+    cy.window().then(window => {
+      assert.isFunction(window.EditorJS, 'window.EditorJS is a function');
+    });
+  });
+
+  it('create editor js instance on the page and check if it is visible', () => {
+    cy.visit(Cypress.env('local_test_server_url'));
+
+    /**
+     * Initialize editor with some paragraph data imported form basic/paragraph_data.json fixture.
+     */
+    cy.fixture('basic/paragraph_data').then((paragraph) => {
+      cy.window().then(window => {
+        const editor = new window.EditorJS({
+          holder: 'editorjs',
+          data: {
```

Comment:
I think it is related to the tests of Editor Config. In this test, you can use an empty config

Reply:
done.